protocol/communities/community.go

```diff
@@ -1409,24 +1409,24 @@ func (o *Community) MarshaledDescription() ([]byte, error) {
 }
 
 func (o *Community) toBytes() ([]byte, error) {
+	if !o.IsOwner() {
+		// This should not happen, as we can only serialize on our side if we
+		// created the community
+		if len(o.config.MarshaledCommunityDescription) == 0 {
+			return nil, ErrNotOwner
+		}
 
-	// This should not happen, as we can only serialize on our side if we
-	// created the community
-	if o.config.PrivateKey == nil && len(o.config.MarshaledCommunityDescription) == 0 {
-		return nil, ErrNotAdmin
-	}
-
-	//We are not owner or admin, use the received serialized version
-	if !o.IsOwnerOrAdmin() {
```

Comment:
admins should also be available to marshal description

Reply:
Admins don't have private key so how can they sign the description?

Reply:
signing will be skipped, I think it is ok. At least if key is empty, we do not return an error